parm/wflow/prep.yaml

```diff
@@ -47,21 +48,18 @@ task_make_orog:
       datadep:
         attrs:
           age: 00:00:00:05
-        text: '{{ task_make_grid.GRID_DIR }}/make_grid_task_complete.txt'
+        value: '{{ task_make_grid.GRID_DIR }}/make_grid_task_complete.txt'
```

Comment:
I think we've also erred in our design by using the key name `value:`: The XML spec refers to what appears between the start and end tags as _content_, so this probably should have been (and could be if we decide to do an overhaul at some point), `content:`.

Reply:
I don't mind getting closer to the Rocoto terminology.